io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -153,15 +153,15 @@ public function channelWriteLines(WritableChannel writableChannel, string[] cont
     }
 }
 
-# Write stream of lines to a channel.
+# Write stream of lines to a channel
```

Comment:
Similar concern as in the previous method regarding the term "stream of lines"

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8